torch/csrc/inductor/proxy_executor.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <ATen/core/ivalue.h>
+#include <c10/macros/Export.h>
+#include <string>
+
+namespace torch {
+namespace aot_inductor {
+
+class TORCH_API ProxyExecutor : public torch::CustomClassHolder {
```

Comment:
BTW, there isn't actually any good reason for this to be a CustomClassHolder, right? All you really need is it to be intrusively refcounted. (Do you even need refcounting on it?)

Reply:
Yes, I need it to be intrusively refcounted, as it's used in FbAOTInductorModel's def_pickle. 

I wonder how to make this intrusively refcounted, but via CustomClassHolder?

Reply:
Why do we pickle the proxy executor 🤔